modules/api_docs/php/module.class.inc

```diff
@@ -0,0 +1,66 @@
+<?php declare(strict_types=1);
+/**
+ * PHP Version 7
+ *
+ * @category Documentation
+ * @package  Main
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ */
+namespace LORIS\api_docs;
+
+/**
+ * Class module implements the basic LORIS module functionality
+ *
+ * @category Documentation
+ * @package  Main
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ */
+class Module extends \Module
+{
+    /**
+     * This module is publicly available to people who are not logged in to LORIS.
+     *
+     * @return bool
+     */
+    public function isPublicModule()
+    {
+        return true;
```

Comment:
I don't think this module should be public. A developer who wants to use it should probably have an account on the server it's being used on, otherwise it seems like it might leak data.

Reply:
anyways demo.loris.ca can make it accessible to anyone

Reply:
done